tests/test.py

```diff
@@ -0,0 +1,38 @@
+import time
+import logging
+import os
+import subprocess
+
+from avocado import VERSION
+from avocado import Test
+
+class MinishiftISOTest(Test):
+
+    def setUp(self):
+        ''' Test Setup '''
+        self.test_DIR = os.path.dirname(os.path.realpath(__file__))
+        self.log.info("################################################################")
+        self.log.info("Avocado version : %s" % VERSION)
+        self.log.info("################################################################")
+
+    def test_minishift_centos_iso_build(self):
+        ''' minishift centos iso build test '''
+        self.log.info("Testing minishift centos iso build ...")
+        process = subprocess.Popen("make", shell=True, stdout=subprocess.PIPE)
```

Comment:
hmm, I think you can assume that the ISO is built when you test. You are effectively building the ISO as part of the test. The build should be something like:

    $ make iso
    $ make test 

By doing so, you also impose an order.